ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -564,26 +565,52 @@ private Optional<List<Request>> extractRequests(final Optional<List<String>> may
                 .collect(Collectors.toList()));
   }
 
-  private void logImportedBlockInfo(final Block block, final int blobCount, final double timeInS) {
+  private void logImportedBlockInfo(
+      final Block block,
+      final int blobCount,
+      final double timeInS,
+      final Optional<Integer> nbParallelizedTransations) {
     final StringBuilder message = new StringBuilder();
-    message.append("Imported #%,d / %d tx");
+    final int nbTransactions = block.getBody().getTransactions().size();
+    message.append("Imported #%,d  (%s)|%5d tx");
     final List<Object> messageArgs =
         new ArrayList<>(
-            List.of(block.getHeader().getNumber(), block.getBody().getTransactions().size()));
+            List.of(
+                block.getHeader().getNumber(),
+                shortenString(block.getHash().toHexString()),
```

Comment:
a major downside to this is not being able to find a block hash on etherscan (e.g. to check it is canonical).

Reply:
My first version doesn't shorten the hash specifically for that reason, But if there is any issue related to the block hash, we still have part of the hash, even if is not like having the whole hash. To have a clear log, and have everything in one line, we need to make choices, and I already shared with other contributors, and seems to have consensus on it. Happy to discuss about it more if needed.

Reply:
I agree about the desirability of the one-liner, which is why I've made brevity suggestions elsewhere. 
I think we should go either go all in on brevity or all in on useful information (or have two logs 😁 ).
This PR is in the middle of the spectrum IMO.

Reply:
> But if there is any issue related to the block hash, we still have part of the hash
I regularly copy and paste this hash into etherscan to get on its canonicality. 
Admittedly this is an exception case, however it occurs frequently.

A non-canonical hash that was produced by only your node will be lost forever, but I suppose we can compare to the hash of the same block number to verify canonicality.